ddpui/models/__init__.py

```
@@ -4,3 +4,4 @@
 from ddpui.models.org_preferences import OrgPreferences
 from ddpui.models.org_supersets import OrgSupersets
 from ddpui.models.org_wren import OrgWren
+import ddpui.models.dbt_automation
```

Ruff (0.8.2)

`ddpui.models.dbt_automation` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)